### PR TITLE
Refactor catalog-ui and instances-ui

### PR DESCRIPTION
**Pull request by magicmatatjahu**
Confirm these statements before you submit your pull request:

- [x] I have read and submitted the required [Contributor Licence Agreements](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#agreements-and-licenses).
- [x] This pull request follows the contributing guidelines, recommended Git workflow, and templates.
- [ ] I have tested my changes.
- [ ] I have updated the relevant documentation.
---

Changes proposed in this pull request:
- Extract generic components from `catalog-ui` and `instances-ui` to `@kyma-project/react-components` library, like a `Table`, `Tooltip`, `Modal` (Information, Confirmation and Steps), `Notification`, `Tabs`, `Forms` (Input, Checkbox, Select), `Dropdown` and etc.
- Refactor `catalog-ui`:
  * Separation logical and presentations components (extract components created with `styled-components` to another file per folder: `styled.js`)
  * Split components like `ServiceClassList` and `ServiceClass` to smaller components,
  * proposition for `import` architecture: first import external libraries, then components from `@kyma-project/react-components`, then components from `components.js` file and at the end functions and utils,
  * Add PropTypes to all components in project.
- Refactor `instances-ui`: similary as above,
- Add Tooltip to modal for creating ServiceBindings (for showing errors),
- Prepare `instances-ui` to show notifications,
- Repair showing content in `Tabs` component,
- Change documentation URL from ServiceClass in `catalog-ui` from read-only string to hiperlink (with attr `target="_blank"`),
- Bump to `0.0.14` version of `@kyma-project/react-components` and publish it.

### CODE REVIEW

File: catalog/src/commons/helpers.js [hunk 1]
```diff
@@ -36,20 +39,51 @@ export const getResourceDisplayName = resource => {
...
-    if (Object.keys(o[k]).length === 0) {
-      delete o[k];
+    clearEmptyPropertiesInObject(object[key]);
+    if (Object.keys(object[key]).length === 0) {
+      delete object[key];
     }
   }
 }
+
+export function compareTwoObjects(obj1, obj2) {
```
Comment (pkosiec) [inline catalog/src/commons/helpers.js]: I wonder if we can use any existing library here

File: catalog/src/components/ServiceClassDetails/components.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  text-align: left;
+  flex: 0 0 auto;
+
+  ${media.phone`
+    width: 100%;
+    flex: 1 1 100%;
+  `};
+`;
+
+// export const ContentWrapper = styled.div`
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/components.js]: Remove all commented code

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/CreateInstanceModal.component.js [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+          minWidth: '261px',
+        },
+      });
+    }
+
+    this.setState({
+      creatingInstance: false,
+    });
+    // false - prevent close modal
+    // true - close modal
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/CreateInstanceModal.component.js]: I think the comment is unnecessary

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+      instanceWithNameAlreadyExists: false,
+      stepFilled: props.data.stepFilled,
+    };
+  }
+
+  componentWillMount() {
+    const { serviceClass } = this.props;
+    const { formData } = this.state;
+
+    if (serviceClass.serviceClass) {
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]: Convert to `if (!serviceClass.serviceClass) return;`

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+  }
+
+  componentDidUpdate(nextProps, nextState) {
+    const {
+      formData,
+      invalidInstanceName,
+      instanceWithNameAlreadyExists,
+    } = this.state;
+
+    if (!compareTwoObjects(this.state, nextState)) {
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]: If the `if` block is too big, that usually means you have to invert the condition in the statement

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+      const stepFilled =
+        nextState.formData.name &&
+        !invalidInstanceName &&
+        !instanceWithNameAlreadyExists;
+
+      this.setState({
+        stepFilled: stepFilled,
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]:
```
stepFilled,
```

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js [hunk 7]
```diff
@@ -0,0 +1,242 @@
...
+
+      const tooltipData = !stepFilled
+        ? {
+            type: 'error',
+            content: 'Fill out all mandatory fields',
+          }
+        : null;
+
+      this.props.callback({
+        formData: formData,
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]:
```
formData,
tooltipData,
firstStepFilled: stepFilled
```

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js [hunk 8]
```diff
@@ -0,0 +1,242 @@
...
+        firstStepFilled: stepFilled,
+        tooltipData: tooltipData,
+      });
+
+      clearTimeout(this.timer);
+      if (!invalidInstanceName) {
+        this.timer = setTimeout(() => {
+          this.checkNameExists(formData.name);
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]: It's good to check after the timeout if the method and formData is still defined - what if user closes the modal within 250ms?

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js [hunk 9]
```diff
@@ -0,0 +1,242 @@
...
+    }
+  }
+
+  componentWillUnmount() {
+    clearTimeout(this.timer);
+  }
+
+  onChangeName = value => {
+    this.setState({
+      firstStepFilled: !value ? false : true,
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]: Simplify it 😛

File: catalog/src/components/ServiceClassDetails/components/Events/components.js [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+`;
+
+export const VersionSpan = styled.span`
+  color: #0a6ed1;
+`;
+
+export const Code = styled.code`
+  width: 100%;
+  white-space: pre-wrap;
+  white-space: -moz-pre-wrap;
```
Comment (pkosiec) [inline catalog/src/components/ServiceClassDetails/components/Events/components.js]: You don't have to prefix CSS, styled-components has autoprefixer out of the box

File: components/react/src/commons/utils.js [hunk 11]
```diff
@@ -0,0 +1,7 @@
+export function copyPropsToState(props, state, focus = false) {
```
Comment (pkosiec) [inline components/react/src/commons/utils.js]: tenor-176794473

What's the purpose of this function? 😄

File: components/react/src/components/Forms/Input/index.js [hunk 12]
```diff
@@ -0,0 +1,198 @@
...
+    if (!validFunctions || validFunctions.length == 0) {
+      return;
+    }
+
+    let results = [],
+      numberOfSucesses = 0;
+    validFunctions.forEach(func => results.push(func(value)));
+
+    for (const result of results) {
+      if (result !== undefined) {
```
Comment (pkosiec) [inline components/react/src/components/Forms/Input/index.js]: Please simplify it (by reversing the condition) - a few nested `if`s are ugly

File: components/react/src/components/Forms/Input/index.js [hunk 13]
```diff
@@ -0,0 +1,198 @@
...
+
+  validate = value => {
+    const { validFunctions } = this.props;
+    if (!validFunctions || validFunctions.length == 0) {
+      return;
+    }
+
+    let results = [],
+      numberOfSucesses = 0;
+    validFunctions.forEach(func => results.push(func(value)));
```
Comment (pkosiec) [inline components/react/src/components/Forms/Input/index.js]: Convert to map

File: components/react/src/components/Forms/Select/components.js [hunk 14]
```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+
+export const SelectWrapper = styled.div`
+  position relative;
+  width: auto;
+  height: auto;
+`;
+
+// &:after {
```
Comment (pkosiec) [inline components/react/src/components/Forms/Select/components.js]: Remove

File: components/react/src/components/Forms/Select/index.js [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+        {label}
+        {required ? <FieldRequired>*</FieldRequired> : ''}
+      </FieldLabel>
+      <SelectWrapper>
+        <SelectField
+          onChange={e => handleChange(e.target.value)}
+          name={name}
+          value={current ? current : ''}
+        >
+          {firstEmptyValue
```
Comment (pkosiec) [inline components/react/src/components/Forms/Select/index.js]: I'd move it to the top (before `return`)

File: components/react/src/components/Modal/Confirmation/Readme.md [hunk 16]
```diff
@@ -0,0 +1,8 @@
+```jsx
+<ConfirmationModal
+    title="lol"
```
Comment (pkosiec) [inline components/react/src/components/Modal/Confirmation/Readme.md]: tenor-185312516

File: components/react/src/components/Modal/Confirmation/index.js [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+    super(props);
+    this.child = React.createRef();
+  }
+
+  handleConfirmation = () => {
+    try {
+      this.props.handleConfirmation();
+      this.child.handleCloseModal();
+    } catch (err) {
+      this.child.handleCloseModal();
```
Comment (pkosiec) [inline components/react/src/components/Modal/Confirmation/index.js]: it's good to at least log this error

File: components/react/src/components/Modal/Information/Readme.md [hunk 18]
```diff
@@ -0,0 +1,7 @@
+```jsx
+<InformationModal
+    title="lol"
```
Comment (pkosiec) [inline components/react/src/components/Modal/Information/Readme.md]: lol

File: components/react/src/components/Modal/Steps/Readme.md [hunk 19]
```diff
@@ -0,0 +1,11 @@
+```jsx
+<StepsModal
+    title="lol"
```
Comment (pkosiec) [inline components/react/src/components/Modal/Steps/Readme.md]: lol

File: components/react/src/components/NotificationMessage/index.js [hunk 20]
```diff
@@ -0,0 +1,82 @@
...
+
+  componentWillReceiveProps = () => {
+    this.setState({ show: true });
+  };
+
+  render() {
+    const { type, title, message } = this.props;
+    const { show } = this.state;
+
+    if (!type || !title || !message) {
```
Comment (pkosiec) [inline components/react/src/components/NotificationMessage/index.js]: There is no way to fulfill this condition, as the propTypes are defined

File: components/react/src/components/NotificationMessage/index.js [hunk 21]
```diff
@@ -0,0 +1,82 @@
...
+      return null;
+    }
+    return (
+      <MessageWrapper>
+        {show ? (
```
Comment (pkosiec) [inline components/react/src/components/NotificationMessage/index.js]: Move it to the top:
```
if (!show) {
return null
{
```

File: components/react/src/components/Table/Readme.md [hunk 22]
```diff
@@ -0,0 +1,34 @@
+```jsx
+<Table title="Example Table" columns={[{
+    name: 'Name',
+    size: 0.3,
+    accesor: el => el.name,
+}, {
+    name: 'Dupa',
+    size: 0.4,
+    accesor: el => el.dupa,
```
Comment (pkosiec) [inline components/react/src/components/Table/Readme.md]: tenor-224174075

File: components/react/src/components/Table/Readme.md [hunk 23]
```diff
@@ -0,0 +1,34 @@
...
+}, {
+    name: 'Lol',
```
Comment (pkosiec) [inline components/react/src/components/Table/Readme.md]: lol

File: components/react/src/components/Table/TableContent/Row/index.js [hunk 24]
```diff
@@ -0,0 +1,33 @@
...
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { columnsData, element } = this.props;
+
+    return (
+      <Grid>
+        {element &&
```
Comment (pkosiec) [inline components/react/src/components/Table/TableContent/Row/index.js]: I'd move it to the top - reverse condition and return null

File: components/react/src/components/Table/TableContent/index.js [hunk 25]
```diff
@@ -0,0 +1,46 @@
...
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { columnsData, elements, loading, notFoundMessage } = this.props;
+
+    return (
+      <TableContentWrapper>
+        {loading ? (
```
Comment (pkosiec) [inline components/react/src/components/Table/TableContent/index.js]: Using ternary operator x2 is too much to still call it a clean code 😛

File: components/react/src/components/Table/TableContent/index.js [hunk 26]
```diff
@@ -0,0 +1,46 @@
...
+class TableContent extends React.Component {
+  static propTypes = {
+    columnsData: PropTypes.arrayOf(PropTypes.object).isRequired,
+    elements: PropTypes.arrayOf(PropTypes.object).isRequired,
+    loading: PropTypes.bool.isRequired,
+    notFoundMessage: PropTypes.string,
+  };
+
+  static defaultProps = {
+    notFoundMessage: 'Not found data',
```
Comment (pkosiec) [inline components/react/src/components/Table/TableContent/index.js]: `Missing data`

File: components/react/src/components/Table/index.js [hunk 27]
```diff
@@ -0,0 +1,66 @@
...
+    columns: PropTypes.arrayOf(PropTypes.object).isRequired,
+    data: PropTypes.arrayOf(PropTypes.object).isRequired,
+    loading: PropTypes.bool,
+    notFoundMessage: PropTypes.string,
+  };
+
+  static defaultProps = {
+    title: 'Example Table',
+    loading: false,
+    notFoundMessage: 'Not found resources',
```
Comment (pkosiec) [inline components/react/src/components/Table/index.js]: `Resources not found`

File: components/react/src/components/Table/index.js [hunk 28]
```diff
@@ -0,0 +1,66 @@
...
+    title: PropTypes.string.isRequired,
+    addHeaderContent: PropTypes.any,
+    columns: PropTypes.arrayOf(PropTypes.object).isRequired,
+    data: PropTypes.arrayOf(PropTypes.object).isRequired,
+    loading: PropTypes.bool,
+    notFoundMessage: PropTypes.string,
+  };
+
+  static defaultProps = {
+    title: 'Example Table',
```
Comment (pkosiec) [inline components/react/src/components/Table/index.js]: Just `Table`

File: components/react/src/components/Table/index.js [hunk 29]
```diff
@@ -0,0 +1,66 @@
...
+    notFoundMessage: 'Not found resources',
+  };
+
+  constructor(props) {
+    super(props);
+  }
+
+  extractHeaderData(columns) {
+    let columnsData = [];
+    columns.forEach(column => {
```
Comment (pkosiec) [inline components/react/src/components/Table/index.js]: Use `map`

File: components/react/src/components/Tabs/index.js [hunk 30]
```diff
@@ -0,0 +1,62 @@
...
+
+  static defaultProps = {
+    defaultActiveTabIndex: 0,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      children: []
+        .concat(...this.props.children)
```
Comment (pkosiec) [inline components/react/src/components/Tabs/index.js]: `[...this.props.children].filter(...)`

File: components/react/src/components/Tabs/index.js [hunk 31]
```diff
@@ -0,0 +1,62 @@
...
+        title: child.props.title,
+        onClick: this.handleTabClick,
+        tabIndex: index,
+        isActive: index === this.state.activeTabIndex,
+      });
+    });
+  };
+
+  renderActiveTab = () => {
+    if (this.state.children[this.state.activeTabIndex]) {
```
Comment (pkosiec) [inline components/react/src/components/Tabs/index.js]: I'd reverse the condition to have the happy path on the left side

File: instances/src/commons/helpers.js [hunk 32]
```diff
@@ -35,3 +35,49 @@ export const getResourceDisplayName = resource => {
 
   return resource.displayName || resource.externalName || resource.name;
 };
+
+export function compareTwoObjects(obj1, obj2) {
```
Comment (pkosiec) [inline instances/src/commons/helpers.js]: I'd really like to use an external solution for that

File: instances/src/components/ServiceInstanceDetails/components/Events/components.js [hunk 33]
```diff
@@ -0,0 +1,18 @@
...
+  font-weight: bold;
+`;
+
+export const VersionSpan = styled.span`
+  color: #0a6ed1;
+`;
+
+export const Code = styled.code`
+  width: 100%;
+  white-space: pre-wrap;
```
Comment (pkosiec) [inline instances/src/components/ServiceInstanceDetails/components/Events/components.js]: Autoprefixer is built-in in styled-components

File: instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js [hunk 34]
```diff
@@ -0,0 +1,208 @@
...
+
+  handleInput = value => {
+    this.setState({ nameServiceBindingUsage: value });
+  };
+
+  handleKind = async value => {
+    this.setState({
+      selectedKind: value,
+      selectedResource: '',
+      usageKindResourcesLoading: value ? true : false,
```
Comment (pkosiec) [inline instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js]: `Boolean(value)`

File: instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js [hunk 35]
```diff
@@ -0,0 +1,208 @@
...
+      usageKindResources: null,
+    });
+
+    if (value) {
```
Comment (pkosiec) [inline instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js]: Reverse condition

File: instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js [hunk 36]
```diff
@@ -0,0 +1,208 @@
...
+  content = () => {
+    const {
+      selectedResource,
+      usageKindResourcesLoading,
+      usageKindResources,
+    } = this.state;
+
+    if (usageKindResourcesLoading) {
+      return <Spinner padding="20px" size="30px" color="rgba(50,54,58,0.6)" />;
+    } else if (usageKindResources) {
```
Comment (pkosiec) [inline instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js]: You don't need else there, as there is a return. Also reverse the condition to simplify the code

**Comment by pkosiec** [comment]
Only the content of the modal should be scrolled - not the whole modal (footer with buttons should be always visible):

screen shot 2018-08-16 at 15 25 22

Please set margins for Move up/down/Delete buttons, as well as the line below "Database extensions to install":
screen shot 2018-08-16 at 15 25 48

**Comment by pkosiec** [comment]
Hmm, all mandatory fields are filled, and the tooltip shows on hover of "Next" button: 
screen shot 2018-08-16 at 15 31 55

**Comment by pkosiec** [comment]
If there is no configuration, it should display a text like before (As far as I remember: "No further configuration is needed.")screen shot 2018-08-16 at 15 39 05

**Comment by pkosiec** [comment]
Before:
screen shot 2018-08-16 at 15 41 24

After:
screen shot 2018-08-16 at 15 40 03


Please revert to previous version with the space between icon and text.

**Comment by pkosiec** [comment]
On ServiceInstances view, the delete icon should be aligned to the right:
screen shot 2018-08-16 at 15 44 41

**Comment by pkosiec** [comment]
I know this is not an issue with the refactoring, but could you please add space between "Service" and "Classes"? Thanks 😄 
screen shot 2018-08-16 at 15 49 38

**Comment by michal-hudy** [comment]
Left border is cropped and title is under `close` button
image

Description of Service Class is bigger thatn before:
image

Before:
image

Design:
image

Different font for links in Service Class overview (this bug exists before refator):
image

**Comment by michal-hudy** [comment]
It would be nice to make this tooltip wider:
image

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js (hunk 4 above)
Comment (magicmatatjahu) [reply catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]: Done

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js (hunk 5 above)
Comment (magicmatatjahu) [reply catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]: Done

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js (hunk 6 above)
Comment (magicmatatjahu) [reply catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]: Done

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js (hunk 7 above)
Comment (magicmatatjahu) [reply catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]: Done

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js (hunk 8 above)
Comment (magicmatatjahu) [reply catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]: Done

File: catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js (hunk 9 above)
Comment (magicmatatjahu) [reply catalog/src/components/ServiceClassDetails/components/CreateInstanceModal/FirstStep.component.js]: Done

File: catalog/src/components/ServiceClassDetails/components/Events/components.js (hunk 10 above)
Comment (magicmatatjahu) [reply catalog/src/components/ServiceClassDetails/components/Events/components.js]: Done

File: components/react/src/components/Forms/Input/index.js (hunk 13 above)
Comment (magicmatatjahu) [reply components/react/src/components/Forms/Input/index.js]: Done

File: components/react/src/components/Forms/Select/components.js (hunk 14 above)
Comment (magicmatatjahu) [reply components/react/src/components/Forms/Select/components.js]: Done

File: components/react/src/components/Forms/Select/index.js (hunk 15 above)
Comment (magicmatatjahu) [reply components/react/src/components/Forms/Select/index.js]: Done

File: components/react/src/components/Modal/Confirmation/index.js (hunk 17 above)
Comment (magicmatatjahu) [reply components/react/src/components/Modal/Confirmation/index.js]: Done

File: components/react/src/components/NotificationMessage/index.js (hunk 20 above)
Comment (magicmatatjahu) [reply components/react/src/components/NotificationMessage/index.js]: Done

File: components/react/src/components/NotificationMessage/index.js (hunk 21 above)
Comment (magicmatatjahu) [reply components/react/src/components/NotificationMessage/index.js]: Done

File: components/react/src/components/Table/TableContent/Row/index.js (hunk 24 above)
Comment (magicmatatjahu) [reply components/react/src/components/Table/TableContent/Row/index.js]: Done

File: components/react/src/components/Table/TableContent/index.js (hunk 25 above)
Comment (magicmatatjahu) [reply components/react/src/components/Table/TableContent/index.js]: Done

File: components/react/src/components/Table/index.js (hunk 27 above)
Comment (magicmatatjahu) [reply components/react/src/components/Table/index.js]: Done

File: components/react/src/components/Table/index.js (hunk 29 above)
Comment (magicmatatjahu) [reply components/react/src/components/Table/index.js]: Done

File: components/react/src/components/Tabs/index.js (hunk 30 above)
Comment (magicmatatjahu) [reply components/react/src/components/Tabs/index.js]: Done

File: components/react/src/components/Tabs/index.js (hunk 31 above)
Comment (magicmatatjahu) [reply components/react/src/components/Tabs/index.js]: Done

File: instances/src/components/ServiceInstanceDetails/components/Events/components.js (hunk 33 above)
Comment (magicmatatjahu) [reply instances/src/components/ServiceInstanceDetails/components/Events/components.js]: Done

File: instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js (hunk 34 above)
Comment (magicmatatjahu) [reply instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js]: Done

File: instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js (hunk 35 above)
Comment (magicmatatjahu) [reply instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js]: Done

File: instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js (hunk 36 above)
Comment (magicmatatjahu) [reply instances/src/components/ServiceInstanceDetails/components/ServiceInstanceBindings/components/CreateBindingModal/SecondStep.component.js]: Done

**Pull request merged**